src/db/sysdb.h

```diff
@@ -572,6 +572,25 @@ errno_t sysdb_apply_default_override(struct sss_domain_info *domain,
                                      struct sysdb_attrs *override_attrs,
                                      struct ldb_dn *obj_dn);
 
+enum search_by_orig_dn_type {
```

Comment:
I'm not fond of adding another enum, sorry, simply because I would prefer to not add many things to the public sysdb API. I would vote for reusing `sysdb_member_type`. Although that enum was first introduced to handle members only, it had since been reused to act as a sysdb object type (and thinking about it, I wouldn't oppose a cleanup patch that would rename member_type to object_type or similar).